qiskit/circuit/instruction.py

```diff
@@ -341,9 +341,9 @@ def inverse(self):
         return inverse_gate
 
     def c_if(self, classical, val):
-        """Add classical condition on register classical and value val."""
-        if not isinstance(classical, ClassicalRegister):
-            raise CircuitError("c_if must be used with a classical register")
+        """Add classical condition on register or cbit classical and value val."""
+        if not isinstance(classical, ClassicalRegister) and not isinstance(classical, Clbit):
```

Comment:
```suggestion
        if not isinstance(classical, (ClassicalRegister, Clbit)):
```

Reply:
Also, can you update the comment above `self.condition = None` in `__init__` to point out that `condition` can now also be `Tuple[Clbit, Int]` or `Tuple[Clbit, Bool]`?

Reply:
Changes done.